doc: Bluetooth qualification (#9705)
nrfconnect/sdk-nrf: doc/nrf/ug_bt_qualification.rst

.. _ug_bt_qualification:

Bluetooth qualification
#######################

.. contents::
   :local:
   :depth: 2

Every product using the Bluetooth® LE technology must be certified (qualified) by Bluetooth Special Interest Group (SIG).
Only when your product is qualified, you will be authorized to do the following:

* Prove that your products satisfy the requirements of the Bluetooth license agreements and specifications.
* Use Bluetooth trademark and logos.
* Sell products that use Bluetooth LE technology.

As a result, your products will be listed and can be promoted on the official Bluetooth SIG page.

Pre-qualified design
********************

.. note::
   A *design* is a specific configuration of hardware or software implementation of adopted Bluetooth specifications.

By using pre-qualified designs of products, you can omit the testing phase during your end product's qualification.

> Nordic Semiconductor uses all available `Bluetooth product types`_. Some of them can be qualified by yourself.

Reviewer: It is more complicated than than. There is no restriction as such who can qualify which product type, as long as they are able to perform all the testing. I suggest you remove these two lines.

All versions of End products, Controller Subsystems (:ref:`ug_ble_controller_softdevice` versions), Host Subsystems, and Components are created and qualified by Nordic Semiconductor by default.
Profile Subsystems (for example, :ref:`ug_bt_mesh` versions) can be qualified by any party.
If you want to use a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_ yet, run tests through the `Profile Tuning Suite (PTS)`_ tool and qualify the product.

Qualification process
*********************

The product qualification cannot be inherited, hence, you must complete this process by yourself.
You can either perform a full qualification process for a new product or edit an existing entry for a product that has already been qualified.

Creating new entry
==================

The only way to qualify your product is to complete the `Bluetooth SIG's Qualification Process`_ under your member company's account.
To check if your company is already a SIG member, go to `Join Bluetooth SIG`_ page and search the Member Directory list.

> 

Reviewer: This is generic description. It can be moved above `Creating new entry` heading.

Log into `Launch Studio`_, the official qualification tool, that will guide you through one of the following processes depending on the state of your product:

* `Qualification process with no required testing`_ - If your complete product uses only pre-qualified End products or Subsystem products.
* `Qualification process with required testing`_ - If your complete product uses a new Bluetooth design or pre-qualified Component products.

Updating existing entry
=======================

Update an existing qualification entry through `Launch Studio`_ in the following cases:

> * You significantly changed an existing product's design so that the testing is required.

Reviewer: Why do you say that? There are two paths in the qualification process. 1. Requires testing 2. Does not require testing. Both of these paths can be used to create a "new entry" or "update existing entry".

Can you make this clear by adding the necessary bullets?

Reviewer: Ok

* Your product can be added as a new model to one of your existing qualifications.

> * You switched to another SDK version for an existing product.

Reviewer: I'm not 100% sure if that is a valid reason to update the existing entry. I'm pretty sure that it is not needed. Please cross-check with Miles.

This update happens at Nordic's design listing (such as [this](https://launchstudio.bluetooth.com/ListingDetails/141213)). Since End Product inherits the design 'as is', as long as they are not using new features and simply pushing firmware updates to update the base subsystems with bug fixes etc., they won't need to add a new row in the product list.

Author: @mileslouissmith Hi Miles, could you please advise on this matter? Thank you!

Author: @mileslouissmith Hi Miles, please take a look at Omkar's comment, thank you.

.. note::

   An existing entry can be edited only by the person that completed the listing.
   If you want to change the entry owner, contact SIG support.

> 

Author: sure, added, thank you

  * That the version you want to use is compatible with versions of other Subsystems you want to keep for your product.
  * That relevant Host and SoftDevice Controller Subsystems implemented in the SDK version you want to use are pre-qualified.
    If yes, check the following:

     * If the version you want to use is fully pre-qualified, and has the same Qualification Design ID (QDID) numbers as the ones currently used for your product, then no further action is required.
     * If the version you want to use has at least one QDID number that is different from the numbers already used for your product, then a new qualification is required.
       Log into the `Launch Studio`_ tool and follow the steps for `Qualification process with no required testing`_ option.

Qualification listing
*********************

The `Bluetooth Qualification Listing`_ search lists all qualified designs.
Use it in the following cases:

* To check if your qualification process succeeded.
* To search for existing designs that can be used in your product.
* To check if you can add your device model to an existing qualification entry.

> Matter certification by inheritance
> ***********************************
> 
> Bluetooth QDIDs may be further used to obtain the Matter certification by inheritence.
> See the :ref:`ug_matter_device_certification_inheritance` section for details.

Reviewer 2: There is no Matter certification by inheritance procedure. However to obrain Matter certification it is required to have certification for depndend transports such as Thread or BLE. Thus to certify Matter product, client needs to provide Bluetooth QDID in Matter certification application. Maybe we should skip this section or have sth like below?

```suggestion
Matter certification
********************

Bluetooth QDIDs may be further used to obtain the Matter certification.
See the :ref:`ug_matter_device_certification_inheritance` section for details.
```

Support
*******

> 

Reviewer: Also, it will be nice to add a link to PRD somewhere in this text. 
https://www.bluetooth.com/download/prd_dpd_resolutions/

Author: As I have no access there, could you please let me know what the exact article title is and what it is about? As far as I understand, end users should familiarize with PRD and DPD documents before they start the qualification process, am I right? If so, should I mention that article somewhere at the beginning of **Qualification process** section? Please advise on that, thank you!

Reviewer: Yes. You are correct. You could say what you mentioned above and give link to these documents.

> For details about the qualification process, see the `Qualifications and listings`_ page, or ask `Bluetooth Qualification Consultants`_ for advice.

Reviewer: ... and they don't have the option to contact Nordic tech support?
```suggestion
For details about the qualification process, see the `Qualifications and listings`_ page, or ask `Bluetooth Qualification Consultants`_ for advice. 

If you have questions regarding Nordic's qualified designs and their use in your end products contact <Nordic tech support>.

```

.. _ug_bt_qualification:

Bluetooth qualification
#######################

.. contents::
   :local:
   :depth: 2

Every product using the Bluetooth® LE technology must be certified (qualified) by Bluetooth Special Interest Group (SIG).
Only when your product is qualified, you will be authorized to do the following:

* Prove that your products satisfy the requirements of the Bluetooth license agreements and specifications.
* Use Bluetooth trademark and logos.
* Sell products that use Bluetooth LE technology.

As a result, your products will be listed and can be promoted on the official Bluetooth SIG page.

Pre-qualified design
********************

.. note::
   A *design* is a specific configuration of hardware or software implementation of adopted Bluetooth specifications.

By using pre-qualified designs of products, you can omit the testing phase during your end product's qualification.

> All `Bluetooth product types`_ are used by Nordic Semiconductor, however, some of them can be qualified by yourself.

Reviewer 3: ```suggestion
All `Bluetooth product types`_ are used by Nordic Semiconductor, and some of them can be qualified by yourself.
```

Reviewer 3: as discussed

> All versions of End products, Controller Subsystems (:ref:`ug_ble_controller_softdevice` versions), Host Subsystems, and Components are created and qualified by Nordic Semiconductor by default.

Reviewer: ```suggestion
Nordic's nRF Connect SDK contains qualified portions of various Bluetooth features (:ref:`ug_ble_controller_softdevice` versions).
```

> All versions of End products, Controller Subsystems (:ref:`ug_ble_controller_softdevice` versions), Host Subsystems, and Components are created and qualified by Nordic Semiconductor by default.

Author: Corrected.

Updating existing entry
=======================

Update an existing qualification entry through `Launch Studio`_ in the following cases:

* You significantly changed an existing product's design so that the testing is required.
* Your product can be added as a new model to one of your existing qualifications.
* You switched to another SDK version for an existing product.

.. note::

   An existing entry can be edited only by the person that completed the listing.
   If you want to change the entry owner, contact SIG support.

Switching to another SDK version
--------------------------------

If you want to switch to another SDK version for an existing product, complete the following steps:

1. Go to the `Nordic Semiconductor Infocenter`_ and search for the SoC or SiP model you use.

#. Navigate to the relevant Compatibility Matrix directory.

#. Open the Bluetooth QDIDs article and, based on the table, confirm the following information:

  a. That relevant Host and SoftDevice Controller Subsystems implemented in the SDK version you want to use are pre-qualified.

>      If yes, check the following:

Reviewer 3: Could we switch the order? Does it matter from the process perspective which one is checked first (line 74 or line 81)? I think it would be better to keep step a and b as close to each other as possible given the sentence structure in the main step above.

.. _ug_bt_qualification:

Bluetooth qualification
#######################

.. contents::
   :local:
   :depth: 2

Every product using Bluetooth® LE technology must be certified (qualified) by Bluetooth Special Interest Group (SIG).
This condition must be met to perform the following actions:

* Demonstrate that your products satisfy the requirements of the Bluetooth license agreements and specifications.
* Use Bluetooth trademark and logos.
* Sell products that use Bluetooth LE technology.

As a result, your products will be listed and can be promoted on the official Bluetooth SIG page.

Pre-qualified design
********************

By using pre-qualified designs, you can omit the testing phase during your product's qualification.

The following product types are used and qualified by Nordic Semiconductor:

* End product
* Controller Subsystem product
* Host Subsystem product
* Profile Subsystem product
* Component product

> All versions of Controller Subsystems (:ref:`ug_ble_controller_softdevice` versions), Host Subsystems and Components created by Nordic Semiconductor are qualified by them by default.

Author: Correct - I rephrased this sentence a little bit.

Some Profile Subsystems (e.g., :ref:`ug_bt_mesh` versions) are qualified as well.

> If you want to use a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_, request the relevant product's qualification by Nordic Semiconductor.

Author: Is this statement correct? To be verified by Miles.

> If you want to use a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_, request the relevant product's qualification by Nordic Semiconductor.

Author: ```suggestion
If you want to use a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_, request the relevant product's qualification by Nordic Semiconductor.
```

Is this statement correct? To be verified by Miles.

> If you want to use a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_, request the relevant product's qualification by Nordic Semiconductor.

Author: Is this statement correct? To be verified by Miles.
If so, how exactly can customers request qualifying a Profile?

Reviewer 4: There are two possibilities there: they could qualify it themselves, or ask if Nordic can do it. It is much more likely that they would have to do it themselves than waiting around for Nordic to think about it.  So I would suggest that we change that sentence a bit.  My suggestion below:
_"If you want to use a Profile Subsystem that is not currently listed by Nordic, you will need to qualify it yourself, typically by running the tests with the PTS test tool available from the Bluetooth SIG."_

Author: I see, corrected, thank you!

> If you want to use a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_ yet, run the tests through the `Profile Tuning Suite (PTS)`_ tool and qualify the needed product by yourself.

Author: corrected

Qualification process
*********************

The product qualification cannot be inherited, hence you must complete this process by yourself.
You can either perform a full qualification process for a new product, or edit an existing entry for a product that has already been qualified.

New entry creation
==================

The only way to qualify your product is to complete the `Bluetooth SIG's Qualification Process`_ under your member company's account.
To check if your company is already a SIG member, go to `Join Bluetooth SIG`_ page and search the Member Directory list.

> Log into `Launch Studio`_, the official qualification tool, that will guide you through one of the following paths:

Reviewer 5: It looks like this is required for both new entry AND existing entry. So maybe move this to a new preceding subsection "Qualification requirements"? You could mention there that the Launch Studio is this and that and that it is required, and in "New entry creation" and "Existing entry update" have just content that refers to the Launch Studio...

Author: Thank you, I see your point :) Let me explain why I wrote it this way. I wanted to keep this document as short and sweet as possible to avoid duplicating basic info from the SIG site. I think that end users will check SIG site first, and then will try to find Nordic-related details on Nordic page. So, yes, some parts are common for both cases, but I wanted to clearly separate new from existing entries sections so that end users can pick up and read only the most crucial parts for them - they know the basics already. If you remain unconvinced, we can discuss it via Teams ;)

Reviewer 5: I became convinced.

* `Qualification process with no required testing`_ - If your complete product uses only pre-qualified End products or Subsystem products.
* `Qualification process with required testing`_ - If your complete product uses a new Bluetooth design or pre-qualified Component products.

.. note::
   A *design* is a specific configuration of hardware or software implementation of adopted Bluetooth specifications.

Existing entry update
=====================

Update an existing qualification entry through `Launch Studio`_ in the following cases:

* You changed an existing product's design significantly so that the testing is required.
* Your product can be added as a new model to one of your existing qualifications.

> * You switched to another SDK version for an existing product.
> 
> If you want to switch to another SDK version for an existing product, perform the following steps:

Reviewer 5: ```suggestion
* You switched to another SDK version for an existing product.

Switching to another SDK version
--------------------------------

If you want to switch to another SDK version for an existing product, complete the following steps:
```
"complete" has most occurrences in the SDK docs in this context, so let's stay consistent.

> 1. Go to the `Nordic Semiconductor Infocenter`_, search for the SoC or SiP model you use, and navigate to the relevant Compatibility Matrix directory.

Reviewer 5: ```suggestion
1. Go to the `Nordic Semiconductor Infocenter`_ and search for the SoC or SiP model you use.
2. Navigate to the relevant Compatibility Matrix page.
```

#. Open the Bluetooth QDIDs article and, based on the table, confirm the following information:

  a. That relevant Host and SoftDevice Controller Subsystems implemented in the SDK version you want to use are pre-qualified.
     If yes, perform one of the following actions:

>      * If the version you want to use is fully pre-qualified, and has the same Qualification Design ID (QDID) numbers as the ones used for your product, then no further action is required.

Reviewer 6: ```suggestion
     * If the version you want to use is fully pre-qualified, and has the same Qualification Design ID (QDID) numbers as the ones currently used for your product, then no further action is required.
```
Adding this suggestion as during the first read it wasn't entirely clear for me which QDIDs to compare.

Author: Fair point, corrected.

#. Open the Bluetooth QDIDs article and, based on the table, confirm the following information:

  a. That relevant Host and SoftDevice Controller Subsystems implemented in the SDK version you want to use are pre-qualified.

>      If yes, perform one of the following actions:

Reviewer 3: ```suggestion
     If yes, check the following:
```

Reviewer 3: Wouldn't say 'perform the following action' since the first bullet says that no action is required. Also, I feel that the main idea for these 2 is to determine further steps based on this check, so this word might be more fitting

>   b. That the version you want to use is compatible with versions of other Subsystems you want to keep for your product.

Author: Lines 62-81 have been corrected and adjusted. Miles, please re-review.

Reviewer 4: That seems ok to me now.

>   b. That the version you want to use is compatible with versions of other Subsystems you want to keep for your product.

Reviewer 3: ```suggestion
#. That the version you want to use is compatible with versions of other Subsystems you want to keep for your product.

     * If the version you want to use is fully pre-qualified, and has the same Qualification Design ID (QDID) numbers as the ones currently used for your product, then no further action is required.
     * If the version you want to use has at least one QDID number that is different from the numbers already used for your product, then a new qualification is required.
       Log into the `Launch Studio`_ tool and follow the steps for `Qualification process with no required testing`_ option.

>   #. That the version you want to use is compatible with versions of other Subsystems you want to keep for your product.

Reviewer 3: tbc - keep or use?

Qualification listing
*********************

`Bluetooth Qualification Listing`_ search lists all qualified designs. Use it in the following cases:

* To check if your qualification process succeeded.
* To search for existing designs that can be used in your product.
* To check if you can add your device model to an existing qualification entry.

> Dependencies

Reviewer 5: ```suggestion
Matter certification by inheritance
```
Maybe let's reference Matter in the title? If not, can we change the header to something else? "Dependencies" can suggest both things that depend on the certification and things that the certification depends on, so I'd rather be more specific.

Matter certification by inheritance
***********************************

Bluetooth QDIDs may be further used to obtain the Matter certification by inheritence.
See the :ref:`ug_matter_device_certification_inheritance` section for details.

Support
*******

> For details about the qualification process, see the `Qualifications and listings`_ page or ask `Bluetooth Qualification Consultants`_ for advice.

Reviewer 3: ```suggestion
For details about the qualification process, see the `Qualifications and listings`_ page, or ask `Bluetooth Qualification Consultants`_ for advice.
```
